maintainers/maintainer-list.nix

```diff
@@ -14236,6 +14236,12 @@
     githubId = 32751441;
     name = "urlordjames";
   };
+  ursi = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.